vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -100,6 +100,8 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.28.0", # for k8s 1.28.x
     # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
     "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.15",
+    # Azure CNI. Owner: rbtr (Evan Baker)
+    "mcr.microsoft.com/containernetworking/azure-cni:v1.5.15",
```

Comment:
What is the difference between this cni  version and the legacy azure cni?

Reply:
this is not a change to the azure-cni itself, it is a new way of packaging it - we want to deliver it consistently in all scenarios via this container